### PR TITLE
feat(dev-server-core): expose websockets server on the manager

### PR DESCRIPTION
**Pull request by jorenbroekema**
## What I did

1. make the webSocketServer property of the WebSocketsManager public, so it can be used by users of Node API for WDS
2. add documentation about this

### CODE REVIEW

File: docs/docs/dev-server/node-api.md [hunk 1]
```diff
@@ -133,6 +133,25 @@ async function main() {
 main();
 ```
 
+## Websockets Server
+
+The WebSocketsManager is exposed in case you need more fine-grained control. It contains three helpers:
+
+- `sendImport`: imports the given path, executing the module as well as a default export if it executes a function
```
Comment (LarsDenBakker) [inline docs/docs/dev-server/node-api.md:140]:
```suggestion
- `sendImport`: imports the given path, executing the module as well as a default export if it exports a function
```

Comment (jorenbroekema) [reply docs/docs/dev-server/node-api.md]: fixed

**Comment by jorenbroekema** [comment]
I'm not sure what could be the reason for a sudden failing test in the test-runner-core, is Saucelabs known to be flaky like this?

I can't really debug this locally without a saucelabs account I think right?

**Comment by LarsDenBakker** [comment]
haven't seen it be flaky, but with a remote service it's possible

**Pull request merged**